### DISCUSSION TITLE
Identify unlisted repos (e.g. repos that haven't been categorized)

### DISCUSSION THREAD

**Discussion by l1qu1d** [community/community#11742] [Other Feature Feedback, Questions, & Ideas]
It's **necessary** to be able to view all the unlisted repos so that we could then categorize them quickly. I've included an image of a possible solution to what the proposed feature could look like.

![example](https://user-images.githubusercontent.com/1037693/154768864-32b06cff-c5a8-437f-ae0a-08abafd65660.png)

**Comment by chikaku** [comment]
Really need this, now I have 900+ repos to be categorized, but put it into `unlisted` list may make it seems like listed for GitHub. Add an `unlisted` filter for show Stars would be better.

**Reply by l1qu1d** [reply]
This would also be an acceptable solution as well. Having a way, in general, to show repos that haven't been categorized is a necessary feature that should've been implemented before beta. I'm surprised this feature request hasn't been responded to by the team yet.

**Reply by silver6633** [reply]
> Really need this, now I have 900+ repos to be categorized, but put it into `unlisted` list may make it seems like listed for GitHub. Add an `unlisted` filter for show Stars would be better.

**Comment by sayanarijit** [comment]
Really need it...

**Comment by rraadd88** [comment]
Yup, much needed.

**Comment by omar-hanafy** [comment]
The new "List" feature is great for organizing my favorite repositories. However, it would be even better if we had the option to filter and view unlisted repos. This would help me keep track of which repos still need to be categorized and make organizing my lists much easier.

**Comment by woden255** [comment]
> It's **necessary** to be able to view all the unlisted repos so that we could then categorize them quickly. I've included an image of a possible solution to what the proposed feature could look like.
> 
> ![example](https://user-images.githubusercontent.com/1037693/154768864-32b06cff-c5a8-437f-ae0a-08abafd65660.png)
> 
> 

This guy is cool. I wholeheartedly agree with this.

**Comment by Truonghau83** [comment]
> It's **necessary** to be able to view all the unlisted repos so that we could then categorize them quickly. I've included an image of a possible solution to what the proposed feature could look like.
> 
> 
> 
> ![example](https://user-images.githubusercontent.com/1037693/154768864-32b06cff-c5a8-437f-ae0a-08abafd65660.png)
> 
> 
> 
>

**Reply by Truonghau83** [reply]
Hai

**Comment by tungnc78** [comment]
Mbf

**Comment by Ethkuil** [comment]
It's 2023 July. I fail to see this feature haven't been provided.

**Comment by ksharizard** [comment]
Hi! I was facing the same issue with having a lot of stars and not knowing which are unlisted. I created a bash script to get a user's stars and lists and find the unlisted stars. I tested this on my profile and it worked well to output the unlisted repos. Just linking it here if anybody finds it interesting and wants to try it out:

https://github.com/ksharizard/github-lists
